ktlint-ruleset-standard/src/main/kotlin/com/pinterest/ktlint/ruleset/standard/internal/trailingcomma/TrailingCommaState.kt

```diff
@@ -0,0 +1,24 @@
+package com.pinterest.ktlint.ruleset.standard.internal.trailingcomma
```

Comment:
See comment on TrailingCommaExtensions. For me it would be fine that bot rules would declare this enum themselves. Technically it can be viewed as duplicated codebecause you can extract it, but functionally they are used to support different rules.